src/Microsoft.TestPlatform.Utilities/InferRunSettingsHelper.cs

```diff
@@ -204,6 +206,16 @@ public static void UpdateDesignMode(XmlDocument runSettingsDocument, bool design
         AddNodeIfNotPresent(runSettingsDocument, DesignModeNodePath, DesignModeNodeName, designModeValue);
     }
 
+    /// <summary>
+    /// Updates the <c>RunConfiguration.BatchSize</c> value for a run settings. Doesn't do anything if the value is already set.
+    /// </summary>
+    /// <param name="runSettingsDocument">Document for runsettings xml</param>
+    /// <param name="batchSizeValue">Value to set</param>
+    public static void UpdateBatchSize(XmlDocument runSettingsDocument, long batchSizeValue)
```

Comment:
It could be nice to return a bool to say if the value was actually set or not.

Reply:
Following the same pattern as other methods on this class so ignoring this remark.